plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -715,6 +715,13 @@ public void testLikePredicatePushdown()
         }
     }
 
+    @Test
+    public void testOrPredicatePushdown()
+    {
+        assertThat(query("SELECT * FROM nation WHERE name = 'ALGERIA' OR regionkey = 4"))
```

Comment:
- two `!=`
- mix of `=` and `!=`
- assert no pushdown in case of a IS NULL comparision
- assert correctness if someone writes `= NULL` or `!= NULL`

Reply:
added